web/Dockerfile

```diff
@@ -10,7 +10,7 @@ COPY docker-entrypoint.sh /usr/local/bin/
 # Install packages and dependencies for postorius and hyperkitty
 # Add user for executing apps, change ownership for uwsgi+django files and set execution rights for management script
 RUN set -ex \
-	&& apk add --no-cache --virtual .build-deps gcc libc-dev linux-headers postgresql-dev \
+	&& apk add --no-cache --virtual .build-deps gcc libc-dev linux-headers postgresql-dev mariadb-dev \
```

Comment:
Is `mysql-dev` not required here for mysql? 
I understand that you have been testing with mariadb, would this library be enough to work with mysql as well?